__snapshots__/flex-recycle.test.js

```diff
@@ -72,38 +89,38 @@ You must be authenticated.
 `
 
 exports['flex recycle by specifying credentials as options when valid and existent serviceId should succeed 1'] = `
-debug:  Checking for package updates
-debug:  Request:  POST http://localhost:3234/session
-debug:  Response: POST http://localhost:3234/session 200
-debug:  Request:  POST http://localhost:3234/v2/jobs
-debug:  Response: POST http://localhost:3234/v2/jobs 200
-info:  Recycle initiated, received job idOfJobThatIsRecyclingTheService
-debug:  Writing JSON to file projectSetupPath
-debug:  Writing contents to file projectSetupPath
-info:  Saved job ID to project settings.
-debug:  Request:  DELETE http://localhost:3234/session
-debug:  Response: DELETE http://localhost:3234/session 204
-debug:  Logged out current user.
+[debug] Checking for package updates
+[debug] Request:  POST http://localhost:3234/session
+[debug] Response: POST http://localhost:3234/session 200
+[debug] Request:  POST http://localhost:3234/v2/jobs
+[debug] Response: POST http://localhost:3234/v2/jobs 200
+[debug] Writing JSON to file projectSetupPath
+[debug] Writing contents to file projectSetupPath
+[debug] Saved job ID to project settings.
+[debug] Request:  DELETE http://localhost:3234/session
+[debug] Response: DELETE http://localhost:3234/session 204
+[debug] Logged out current user.
+Recycle initiated. Job: idOfJobThatIsRecyclingTheService
 
 `
 
 exports['flex recycle by specifying credentials as options when valid and non-existent serviceId should fail 1'] = `
-debug:  Checking for package updates
-debug:  Request:  POST http://localhost:3234/session
-debug:  Response: POST http://localhost:3234/session 200
-debug:  Request:  POST http://localhost:3234/v2/jobs
-debug:  Response: POST http://localhost:3234/v2/jobs 404
-debug:  Request:  DELETE http://localhost:3234/session
-debug:  Response: DELETE http://localhost:3234/session 204
-debug:  Logged out current user.
-error:  DataLinkNotFound: The specified data link could not be found.
+[debug] Checking for package updates
+[debug] Request:  POST http://localhost:3234/session
+[debug] Response: POST http://localhost:3234/session 200
+[debug] Request:  POST http://localhost:3234/v2/jobs
+[debug] Response: POST http://localhost:3234/v2/jobs 404
+[debug] Request:  DELETE http://localhost:3234/session
+[debug] Response: DELETE http://localhost:3234/session 204
+[debug] Logged out current user.
+DataLinkNotFound: The specified data link could not be found.
```

Comment:
I find the 'DataLinkNotFound' part easily searchable (if at some point you want to search in a log file).
Personally, I'd prefer:
`[error] DataLinkNotFound: The specified data link could not be found.`

I think it would be better this to be decided now (given the upcoming release).

Reply:
I am OK with having "[error]" as a prefix. And on second thought, the "message" of the error can sometimes bring more and a bit different information than the error type ("DataLinkNotFound"). So, I am OK with your preferred format.